### PR TITLE
FIxed some stuff from trello

### PR DESCRIPTION
**Pull request by Swedengamer100**
- Removed dungeon related stuff from ornate zombie sword

- Set bonus intel as 0 on crab hat (dunno if that's the right change but it looks better then just removing the whole line)

- Fixed Hunk of Blue ice rarity to english

- Added gifts to parent, from red to white

- Fixed Turbo-potato req

- Removed dungeon stats from Midas Staff. Changed price paid and bonus to 0 for both sword and staff

### CODE REVIEW

**Comment by TymanWasTaken** [comment]
I already did 5/6 of those but ok #399

**Comment by Swedengamer100** [comment]
welp

**Comment by IRONM00N** [comment]
Please resolve conflicts

File: items/MIDAS_STAFF.json [hunk 1]
```diff
@@ -26,15 +25,16 @@
     "§7is §316000 §7if the bid was",
     "§7§6100,000,000 Coins §7or higher!",
     "",
+    "§7Price paid: §60 Coins",
+    "§7Base Ability Damage Bonus: §30",
+    "",
     "§7§8This item can be reforged!",
-    "§7§4❣ §cRequires §aCatacombs Level 25",
-    "§a§cwhile in Catacombs",
```
Comment (IRONM00N) [inline items/MIDAS_STAFF.json:31]: Can you add the reqs back, other than that it looks good

Comment (Swedengamer100) [reply items/MIDAS_STAFF.json:31]: But you don't need cata lvl unless it's stared?

Comment (IRONM00N) [reply items/MIDAS_STAFF.json:31]: Oh okay

**Pull request merged**